### PR TITLE
chore: add tests for empty tool descriptions

### PR DESCRIPTION
**Pull request by adubovik**
Analogous to https://github.com/epam/ai-dial-adapter-bedrock/pull/348, but without changes to the business logic. 
Only integration tests were added.

### CODE REVIEW

File: tests/integration_tests/test_chat_completion_generation.py [hunk 1]
```diff
@@ -596,21 +597,41 @@ async def test_tool_call_with_schema_references(chat: Chat):
...
+    "deployment",
+    select(pred(supports_tools), deployments),
+    ids=display_deployment,
+)
+@pytest.mark.parametrize("stream", [True], ids=["stream"])
+@pytest.mark.parametrize(
+    "description", ["", " \n\t", None], ids=["empty", "whitespace", "missing"]
+)
+async def test_tool_call_with_vacuous_description(
+    description: str | None, chat: Chat
```
[the remark this answers is not in the document]
Comment (adubovik) [reply tests/integration_tests/test_chat_completion_generation.py:618]: 👎

**Comment by adubovik** [comment]
/deploy-review
>GitHub actions run: [19369773874](https://github.com/epam/ai-dial-ci/actions/runs/19369773874)
>Environment URL: [review-environment](https://chat-ai-dial-adapter-vertexai-pr-324.gke.test.dial.parts) | [pipeline](https://gitlab.deltixhub.com/Clients/dial/test/review/-/pipelines/723948)
>Chat E2E tests status: [success](https://github.com/epam/ai-dial-ci/actions/runs/19369773874)
>Overlay E2E tests status: [success](https://github.com/epam/ai-dial-ci/actions/runs/19369773874)

**Review by sdryapko — APPROVED** [review]
